pyriemann/utils/utils.py

```diff
@@ -100,3 +100,37 @@ def check_weights(weights, n_weights, *, check_positivity=False):
 
     weights /= np.sum(weights)
     return weights
+
+
+def check_metric(metric, expected_keys=["mean", "distance"]):
```

Comment:
We can reuse this method in `pyRiemann-qiskit`, but we will still need to check the prefix of the string to distinguish between `pyRiemann-qiskit` and `pyRiemann` distances.

I wonder if we can make this method more generic so it can be reused "as it" in `pyRiemann-qiskit`.

May be a `prefix` argument or something like this?

Reply:
I prefer to add a new function `check_qmetric` in pyRiemann-qiskit

`def check_qmetric(metric, expected_keys=["mean", "distance"])`

which then calls `is_qfunction(metric)` and `check_metric(metric, expected_keys= expected_keys)`

Reply:
Ok for me